cowprotocol/cowswap
# 574 Favourite tokens in selector

# Summary
Part of #574 

## Screenshots

### When > 12 favourited tokens in web view
<img width="560" alt="image" src="https://user-images.githubusercontent.com/21335563/171631993-59e8a261-8f43-4512-8bc4-fa21ede09254.png">

### When default or < 12 tokens favourited (note no blur, no extra padding)
<img width="529" alt="image" src="https://user-images.githubusercontent.com/21335563/171632386-e6181300-dda6-4749-b033-1e3e75d13576.png">

### Tooltip
<img width="508" alt="image" src="https://user-images.githubusercontent.com/21335563/171632204-b9d97c39-d291-448d-a9fd-a9810f5afc4b.png">

### Mobile view unaffected
<img width="691" alt="image" src="https://user-images.githubusercontent.com/21335563/171632668-b60ddbda-2fde-4433-97ff-829fe98118c9.png">

## QA
1. head to `/account/tokens`
2. add a bunch of tokens
3. > 12 
4. check web view token selector
5. check mob view token selector
6. be happy (hopefully)
